src/nectarchain/dqm/pixel_participation.py

```diff
@@ -0,0 +1,148 @@
+from dqm_summary_processor import DQMSummary
+from matplotlib import pyplot as plt
+from ctapipe.visualization import CameraDisplay
+from ctapipe.instrument import CameraGeometry
+from ctapipe.coordinates import EngineeringCameraFrame
+import numpy as np
+
+
+class PixelParticipation_HighLowGain(DQMSummary):
+    def __init__(self, gaink):
+        self.k = gaink
+        return None
+
+    def ConfigureForRun(self, path, Pix, Samp, Reader1):
+        # define number of pixels and samples
+        self.Pix = Pix
+        self.Samp = Samp
+
+        self.counter_evt = 0
+        self.counter_ped = 0
+
+        self.camera = CameraGeometry.from_name("NectarCam-003").transform_to(EngineeringCameraFrame())
```

Comment:
Please try not to hardcode the camera geometry version, as e.g. done in other classes.

Reply:
Done